components/collaboration-card/card-components/CollaborationDetails.tsx

```
@@ -80,59 +92,61 @@ const CollaborationDetails: FC<CollaborationDetailsProps> = ({
+                chipText={content}
+                chipIcon={
+                  content === "Instagram"
+                    ? faInstagram
+                    : content === "Facebook"
+                      ? faFacebook
+                      : content === "Youtube"
+                        ? faYoutube
+                        : faInstagram
+                }
+              />
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)